analyzer/evmabibackfill/evm_abi_backfill.go

```diff
@@ -167,134 +168,102 @@ func cleanTxRevertReason(raw string) ([]byte, error) {
 	return base64.StdEncoding.DecodeString(s)
 }
 
+func (p *processor) processEvent(ev *abiEncodedEvent, contractAbi abi.ABI) (*string, []*abiEncodedArg, *ethCommon.Hash) {
```

Comment:
For this and the next two funcs: Can you document the return values (the types are not very descriptive), or use named return params?

Optional: My instinct is to name these `parseX` rather than `processX`, as I think we normally use `processX()` for "parse + enqueue db statements". But either works; your call.

Reply:
Thanks! Added docstrings, and also renamed the methods